en/docs/learn/authentication-session-persistence.md

```diff
@@ -64,51 +64,51 @@ into either the service provider or the WSO2 Identity Server.
             However, if you have persistence, you can rely on it as well. This
             increases the reliability of the overall system.
 
-In WSO2 Identity Server has multiple local caching usage in different
+WSO2 Identity Server has multiple local caching usage in different
 layers to improve the product performance. But most of the caches are
 ready to persist in the database if it is required.
 
 There are three type of data objects that are persisted in the database.
 
 #### 1. Session Data
 
-Once the user get authenticated over WSO2 Identity Server, it will
-create a Session data object which is stored the authenticated user and
-the other authentication flow details. This will store in the database
-to share across the cluster nodes.
+Once the user gets authenticated over WSO2 Identity Server, it will
+create a session data object which stores the authenticated user and
+the other authentication flow details. This will be stored in the database
+to be shared across the cluster nodes.
 
 #### 2. Operational Data
 
-This is covered the same session data as in above but the outdated
-records. Ex: Once user get authenticated, there will be a record for
-login status to that session id. Then again that user get logout from
+This covers the same session data as in above but the outdated
+records. Ex: Once user gets authenticated, there will be a record for
+login status to that session id. Then again when that user is logged out from
 the system, we are not removing the above record from the table and
 instead of that we add a new record to the same session id with status
 called logout. So the valid record is the last one and all the other
 records under that session id will be outdated. Those outdated records
-are belong to the Operational Data.
+belong to the Operational Data.
 
 #### 3. Temporary Data
 
-In authentication flow, there are many temporary data object that will
+In authentication flow, there are many temporary data objects that will
```

Comment:
there are many temporary data objects that are kept for a few seconds only.

Reply:
Changed